packages/mdc-textfield/index.js

```diff
@@ -75,7 +78,37 @@ export class MDCTextfield extends MDCComponent {
           label.classList.remove(className);
         }
       },
-    }, this.getInputAdapterMethods_(), this.getHelptextAdapterMethods_()));
+    }, this.getInputAdapterMethods_(), this.getHelptextAdapterMethods_(), this.getBottomLineAdapterMethods_()));
```

Comment:
I don't know. Maybe for help text since it's more optional than a bottom line. There won't be textfields without a bottom line though. Only `textarea` components don't have it